ramls/loan.json

```
@@ -12,6 +12,38 @@
       "description": "ID of the patron the item was lent to. Required for open loans, not required for closed loans (for anonymization).",
       "type": "string"
     },
+    "borrower": {
+      "description": "Copy of some requesting patron metadata (used for searching and sorting), will be taken from the user referred to by the requesterId",
+      "readonly": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "ID of the patron the item was lent to. Required for open loans, not required for closed loans (for anonymization).",
```

Was this description taken from the `userId` property?

If the loan has been closed and anonymized, then these additional properties will not be available.

```suggestion
          "description": "ID of the borrower",
```

---

This field was added to move all user-related fields to the "borrower" object and essentially it is a copy of the userId field dirctly available in a loan. Do you think it should be removed?

---

I think the ID doesn't need to be included in the `borrower` object, so it can be removed.

---

Done